services-camel/src/test/java/edu/unc/lib/cdr/routing/MetaServicesRouterTest.java

```diff
@@ -207,6 +218,19 @@ public void testProcessSolrFail() throws Exception {
         assertMockEndpointsSatisfied();
     }
 
+    @Test
```

Comment:
so you added this new test case?

Reply:
It was on fcrepo4, i just pulled it over